### ISSUE TITLE
more instructions for custom build

### ISSUE THREAD

**Issue by IngmarBitter** [be5invis/Iosevka#764]
I just discovered Iosevka and love especially the idea of cherry-picking my own characters. 
I was hoping I could clone the repo, paste the customization of 'font-feature-settings' selected on the website build and be done.

Here are the issues I encountered, that could be fixed in part via better documentation:
On Windows 10 I ran in a powershell: `npm run build -- ttf::iosevka`
there were a few missing libraries, but a nice message which command-line would install them. 
After those were fixed I got:
```
> iosevka@4.0.0 build D:\code\FontsIosevka\Iosevka
> verda -f verdafile.js "ttf::iosevka"


Error: Cannot find module 'which'
Require stack:
- D:\code\FontsIosevka\Iosevka\verdafile.js
- D:\code\FontsIosevka\Iosevka\node_modules\verda\lib\cli\verda.js
- D:\code\FontsIosevka\Iosevka\node_modules\verda\bin\verda
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (D:\code\FontsIosevka\Iosevka\verdafile.js:9:15)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Module.require (internal/modules/cjs/loader.js:1019:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\code\\FontsIosevka\\Iosevka\\verdafile.js',
    'D:\\code\\FontsIosevka\\Iosevka\\node_modules\\verda\\lib\\cli\\verda.js',
    'D:\\code\\FontsIosevka\\Iosevka\\node_modules\\verda\\bin\\verda'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! iosevka@4.0.0 build: `verda -f verdafile.js "ttf::iosevka"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the iosevka@4.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ingmar\AppData\Roaming\npm-cache\_logs\2020-12-01T15_30_11_456Z-debug.log
```
Next I installed the latest version of docker and tried: `docker run -it -v $(pwd):/build avivace/iosevka-build` 
I got:
```
docker: image operating system "linux" cannot be used on this platform.
```

So I installed VirtualBox + Ubuntu 20.04 + clone from Iosevka repo
I again had some missing libraries. This time when `verda` was not found there was no message on how to install it. According to the other ones this should have worked: `sudo apt install verda`. But there was no package verda. A web search for Linux + verda was not useful either.

So I installed docker and tried: `sudo docker run -it -v \$(pwd):/build avivace/iosevka-build`
That worked, but made the fonts that I could have just downloaded from the website.
Next I renamed the sample customization file `private-build-plans.sample.toml` to `private-build-plans.toml` and expected I'd now get some custom fonts when again running `sudo docker run -it -v \$(pwd):/build avivace/iosevka-build`
That actually worked! I somehow ended up with two dist directories, one parallel to the repo clone and one inside it. It first looked like the custom .toml had no effect, but it did for the dist folder inside the repo.

Then inside the `private-build-plans.toml` I replaced the name of the plan and ith inherits item from "ss01" to "ss12", I got font directories in the ./dist folder that matched my new name, and I even got the `ubuntu mono style` letters.

Finally, I looked for a place to put the cherry-pick results from the web in the .toml file, but could not find anything that matched. 
After a bit I found that the choice of letter I like has a hover over tooltip with the customization code that I need in the .toml file. Now I can get the customization that I want. But it was a little bumpy to get there. And it would be nice to get there without the need of docker inside a virtual machine. 

So in summary, getting a first time user to be able to build according to his desired cherry-picked choices could be a lot smoother.  Updating the info in the Readme accordingly would be good.

**Comment by be5invis** [comment]
The single command `npm install` should install all the packages. You do *not* install them one by one. README is updated to further clarify the confusing.
For creating `private-build-plans.toml`, this is the right place: https://typeof.net/Iosevka/customizer

**Comment by alexmyczko** [comment]
https://mentors.debian.net/package/fonts-iosevka/

you are welcome

**Comment by AndydeCleyre** [comment]
FWIW I'm [using github as a build server](https://github.com/andydecleyre/archbuilder_iosevka) for iosevka.

**Comment by dixius99** [comment]
> The single command `npm install` should install all the packages. You do _not_ install them one by one. README is updated to further clarify the confusing.

All this time I've been doing `npm install sermver patel spiro stylus` etc. for all the missing packages. This is great to know!

**Comment by 11p5G** [comment]
I builded my own version of Iosevka (v4.4.0) today on linux and no problem was found.
All instructions can be found here: https://github.com/be5invis/Iosevka/blob/master/README.md

If you paste a configuration here, I will build it.

**Comment by alexmyczko** [comment]
Can you turn off your internet and then build it?

**Comment by weiss-d** [comment]
Hi, @be5invis !
I'm trying to make a custom build with Docker. I've made `private-build-plans.toml` via the Customizer.
Could you please specify the command - how should I pass this file to a container?

**Comment by dok42** [comment]
@weiss-d 
You have to run the docker image from where your `private-build-plans.toml` is located.

For example
```
D:\
➜ cd .\Iosevka-Custom\
Iosevka-Custom
➜ ls

    Directory: D:\Iosevka-Custom

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---          22/03/2021    11:18              0 private-build-plans.toml

Iosevka-Custom
➜ docker run -it -v $(pwd):/build avivace/iosevka-build
```

Here I'm inside a directory (D:\Iosevka-Custom) where I did put my `private-build-plans.toml`.
I run the docker image from here.

**Comment by weiss-d** [comment]
@dok42 thanks a lot! now everything works
